build.gradle

```diff
@@ -213,39 +193,19 @@ sonarqube {
     property "sonar.coverage.jacoco.xmlReportPaths", "${project.buildDir}/reports/jacoco/test/jacocoTestReport.xml"
     property "sonar.pitest.mode", "reuseReport"
     property "sonar.pitest.reportsDirectory", "build/reports/pitest"
-    property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/ccd/document/am/*Application.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/util/*Swagger2SpringBoot.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/advice/*ErrorConstants.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/advice/exception/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/enums/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/apihelper/**"
+    // FIXME : RDM-11309
+    property "sonar.exclusions", "src/main/java/uk/gov/hmcts/reform/ccd/document/am/*Application.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/util/*Swagger2SpringBoot.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/advice/*ErrorConstants.java," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/controller/advice/exception/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/configuration/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/enums/**," + "src/main/java/uk/gov/hmcts/reform/ccd/document/am/apihelper/**," + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/service/impl/DocumentManagementServiceImpl.java," + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/model/StoredDocumentHalResource.java," + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/util/ResponseHelper.java," + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/apihelper/Constants.java," + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmController.java" + "src/main/java/uk/gov/hmcts/reform/ccd/documentam/configuration/*.java,src/main/java/uk/gov/hmcts/reform/ccd/documentam/controller/endpoints/CaseDocumentAmController.java"
```

Comment:
some of the formatting is a little out on last two lines - keep consistent with one exclusion per line ?

Reply:
I've tried multi line... but it didn't work. Will see if i can format in a better way.